edit/storage.k8s.io.storageclass/provisioners/kubernetes.io/aws-ebs.vue

```diff
@@ -69,9 +70,11 @@ export default {
       }
     ];
 
-    this.$set(this.value.parameters, 'type', this.value.parameters.type || volumeTypeOptions[0].value);
-    this.$set(this.value.parameters, 'encrypted', this.value.parameters.encrypted || encryptionOptions[0].value);
-    this.$set(this.value.parameters, 'iopsPerGB', this.value.parameters.iopsPerGB || '0');
+    if (this.mode !== _VIEW) {

```

Comment:
Little bit confused why this is needed. Looks like create, clone and view (config) all work fine without.

Reply:
> Storage went to v1 and deprecated beta in k8s 1.6. I didn't easily find exactly what version it was removed in but it's certainly before any current version dashboard supports, so you don't need to set the beta annotation.

@vincent99 ember ends up setting and seemingly reading both which ends up with incorrect behavior when they're set on dashboard but read on ember. Is it harmful to keep this in? I'd rather leave it in until ember is deprecated, I can create an issue or leave a todo.

Reply:
I decided to switch this and all of the other providers to `if (this.mode === _CREATE)`. The reason for this is because on the details page you can still call actions and we don't want to modify the rest of the model when we're just setting some annotations for it to be default. It's effectively applyDefaults() on a sub section of the model.

I specifically noticed this was a problem because if the ebs parameters get modified the backend complains.